modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -715,52 +720,58 @@ class Editor implements EditorObservable {
    * @param {Object} args Optional content object, this gets passed around through the whole save process.
    * @return {String} HTML string that got set into the textarea/div.
    */
-  public save(args?: any): string {
+  public save(args: Partial<EditorContent.GetContentArgs> = {}): string {
     const self = this;
-    let elm = self.getElement(), html, form: HTMLFormElement;
+    let elm: HTMLElement | null = self.getElement();
 
     if (!elm || !self.initialized || self.removed) {
-      return;
+      return '';
     }
 
-    args = args || {};
-    args.save = true;
+    const getArgs: Partial<EditorContent.GetContentArgs> = {
```

Comment:
More changes here to prevent mutation of the passed arguements.

Reply:
And once again certain ways to read the API definition thwart your attempts to improve the code 😩 

`* @param {Object} args Optional content object, this gets passed around through the whole save process.`

Although in this case I'm tempted to leave it, easy to swap back if someone has a really good reason to need the actual object instance to be passed around.

Reply:
I disagree here... this is no way prevents the args being passed. They are still being passed through the whole process.